arkane/kinetics.py

```diff
@@ -71,34 +71,28 @@ def __init__(self, reaction,
                  sensitivity_conditions=None):
         self.usedTST = False
         if Tmin is not None:
-            self.Tmin = quantity.Quantity(Tmin)
+            self.Tmin = Tmin
         else:
-            self.Tmin = None
+            self.Tmin = (298, 'K')
```

Comment:
Could you simplify these assignments into:
`self.Tmin = Tmin if Tmin is not None else (298, 'K')`?
Same for Tmax?

We could also simplify Tcount:
`self.Tcount = Tcount if Tcount > 3. else 50`

Reply:
Good idea. I just added that.